.github/workflows/protoc.yaml

```diff
@@ -0,0 +1,16 @@
+name: protoc
+
+on:
+  pull_request:
+    branches: [ '*' ]
+
+jobs:
+  test-protoc:
+    runs-on: ubuntu-latest
+    steps:
+    - name: checkout source
+      uses: actions/checkout@v2
+
+    - name: generate go libs using protoc
+      run: |
+        make build
```

Comment:
can we check that pushed*.go files are not having any changes once we do make the build here?

Reply:
Done